src/CitMovie.Data/DataContext.cs

```diff
@@ -28,5 +29,23 @@ public class DataContext : DbContext
     
     public DataContext(DbContextOptions<DataContext> options) 
         : base(options) { }
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder) {
+        if (!optionsBuilder.IsConfigured)
+        {
+            // var configuration = new ConfigurationBuilder()
+            //     .SetBasePath(Directory.GetCurrentDirectory())
+            //     .AddJsonFile("appsettings.json")
+            //     .AddEnvironmentVariables()
+            //     .Build();
+            //
+            // optionsBuilder.UseNpgsql(configuration.GetConnectionString("PostgresConnection"));
+        }
+    }
+
+    protected override void OnModelCreating(ModelBuilder modelBuilder)
+    {
+        modelBuilder.Entity<MediaProductionCompany>()
+            .HasKey(od => new { od.MediaId, od.ProductionCompanyId });
```

Comment:
Is this necessary when we use the attributes in the classes for entities?

Reply:
Hmm not sure how other way to make a composite key in attributes
just adding Key, to both?

Reply:
From what I was reading now it seems that this is the way to do it as Key can be used once only, We can make the other one Unique but I guess it is not the same or? apparently it is a workaround by Chat :D

Reply:
https://learn.microsoft.com/en-us/ef/core/modeling/keys?tabs=data-annotations#configuring-a-primary-key

```csharp
[PrimaryKey(nameof(State), nameof(LicensePlate))]
internal class Car
{
    public string State { get; set; }
    public string LicensePlate { get; set; }

    public string Make { get; set; }
    public string Model { get; set; }
}
```

Reply:
https://www.entityframeworktutorial.net/code-first/key-dataannotations-attribute-in-code-first.aspx

It is fine like you did I think :)